echopype/calibrate/cal_params.py

```diff
@@ -202,61 +296,211 @@ def get_vend_cal_params_power(beam: xr.Dataset, vend: xr.Dataset, param: str) ->
     return da_param.where(~transmit_isnull, np.nan)  # set the nan elements back to nan
 
 
-def get_param_BB(
-    vend: xr.Dataset,
-    varname: str,
-    freq_center: xr.DataArray,
-    cal_params_CW: Dict[str, xr.DataArray],
-) -> xr.DataArray:
+def get_cal_params_AZFP(beam: xr.DataArray, vend: xr.DataArray, user_dict: dict) -> dict:
     """
-    Get broadband gain or angle factor for calibrating complex samples.
-
-    Interpolate gain or angle factor in the Vendor_specific group to the center frequency
-    of each ping for BB mode samples if nominal frequency is within the calibrated frequency range
+    Get cal params using user inputs or values from data file.
 
     Parameters
     ----------
-    vend : xr.Dataset
-        Vendor_specific group in an EchoData object
-    varname : str
-        the desired parameter
-    freq_center : xr.DataArray
-        center frequency of the transmit BB signal
-    cal_params_CW : dict
-        a dictionary storing CW calibration parameters as xr.DataArray
+    echodata : EchoData
+        An EchoData object containing data to be calibrated
```

Comment:
I don't see a reference to echodata anywhere here. There should be a reference to `beam`, `vend`, and `user_dict`.

Reply:
Good catch. Done!